src/App.tsx

```diff
@@ -1,173 +1,77 @@
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+import { Loader } from './components/Loader';
+import { NavBar } from './components/NavBar';
+import { PeoplePage } from './pages/PeoplePage';
+import { HomePage } from './pages/HomePage';
+import { PageNotFound } from './pages/PageNotFound';
+import { getPeople } from './api';
+import { Person } from './types';
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const recivePeople = useCallback(
+    async () => {
+      setIsLoading(true);
+      const data = await getPeople();
+
+      setPeople(data);
+      setIsLoading(false);
+    }, [],
+  );
+
+  useEffect(() => {
+    recivePeople();
+  }, []);
+
+  return (
+    <div data-cy="app">
+      <NavBar />
+
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="home" element={<Navigate to="/" replace />} />
+
+            {!isLoading && (
+              <Route path="/people">
+                <Route
+                  index
+                  element={<PeoplePage people={people} />}
+                />
+                <Route
+                  path="/people/:personSlug"
+                  element={<PeoplePage people={people} />}
+                />
+              </Route>
+            )}
+
+            <Route path="*" element={<PageNotFound />} />
+          </Routes>
+
+          <div className="block">
```

Comment:
move it to the `PeoplePage`

Reply:
But it's an external tag. How I can use it inside child's tag?